config/locales/user_mailer/es.yml

```diff
@@ -15,6 +15,10 @@ es:
     password_changed:
       help: NOT TRANSLATED YET
       intro: NOT TRANSLATED YET
+    phone_changed:
+      help: NOT TRANSLATED YET
```

Comment:
seems like we tossed the Spanish translation for the SMS -- could it be re-used here?